src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+
+spring.cloud.gcp.sql.database-name=hyperlocal
+spring.cloud.gcp.sql.instance-connection-name=speedy-anthem-217710:us-central1:hyperlocal
+#Change this to point to your file:<your_credentials_file_path> 
+spring.cloud.gcp.credentials.location=file:/usr/local/google/home/dikshamodi/hyperlocal-gpay-spot/src/main/resources/cred.json
```

Comment:
use relative path, Dont hard code dikshamodi in content

Reply:
Ok. Doing that. Thanks a lot!! The PR got closed by accident. I'll open it again